runtime/interpreter/value.go

```diff
@@ -11541,14 +11541,11 @@ func NewPublicKeyValue(
 		sema.PublicKeyVerifyPoPFunction: publicKeyVerifyPoPFunction,
 	}
 
-	// Validate the public key, and initialize 'isValid' field.
-
-	publicKeyValue.SetMember(
-		interpreter,
-		getLocationRange,
-		sema.PublicKeyIsValidField,
-		validatePublicKey(interpreter, getLocationRange, publicKeyValue),
-	)
+	// Validate the public key.
+	isValid := validatePublicKey(interpreter, getLocationRange, publicKeyValue)
+	if !isValid {
+		panic("invalid public key")
+	}
```

Comment:
This is the big change so bugs are most likely here, or as a result of this change.

Reply:
@SupunS What should it contain? I'm giving it the `ArrayValue` but I'm not sure that's the best information to put there. (change pushed)